tests/unit/multivariate/test_gaussian.py

```diff
@@ -3,41 +3,94 @@
 
 import numpy as np
 import pandas as pd
-
 from copulas import get_qualified_name
 from copulas.multivariate.gaussian import GaussianMultivariate
 from copulas.univariate import GaussianUnivariate
 
 
 class TestGaussianMultivariate(TestCase):
-
     def setUp(self):
-        """Defines random variable to use on tests. """
+        """Defines random variable to use on tests."""
 
         self.data = pd.DataFrame({
             'column1': np.array([
-                2641.16233666, 921.14476418, -651.32239137, 1223.63536668,
-                3233.37342355, 1373.22400821, 1959.28188858, 1076.99295365,
-                2029.25100261, 1835.52188141, 1170.03850556, 739.42628394,
-                1866.65810627, 3703.49786503, 1719.45232017, 258.90206528,
-                219.42363944, 609.90212377, 1618.44207239, 2323.2775272,
-                3251.78732274, 1430.63989981, -180.57028875, -592.84497457,
+                2641.16233666,
+                921.14476418,
+                -651.32239137,
+                1223.63536668,
+                3233.37342355,
+                1373.22400821,
+                1959.28188858,
+                1076.99295365,
+                2029.25100261,
+                1835.52188141,
+                1170.03850556,
+                739.42628394,
+                1866.65810627,
+                3703.49786503,
+                1719.45232017,
+                258.90206528,
```

Comment:
I hate this

Reply:
The only way to disable this is to use to do something like this:
```
# fmt: off
f(
 a, b, c,
 d, e, f,
)
# fmt: on
```
- Personally, I think it's annoying to have to worry about this and would just let ruff handle it

Reply:
Yeah it's not a big deal. I just had to voice my distaste haha